README.md

```diff
@@ -17,14 +17,12 @@ To set your custom new tab url:
 Testing unsigned extensions only works with [non-release](https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/jpm#Install_a_different_version_of_Firefox) [builds](https://wiki.mozilla.org/Add-ons/Extension_Signing). To install the unsigned extension:
 
 - Open Firefox Developer Edition
-- Install the [Extension Auto-Installer](https://addons.mozilla.org/en-US/firefox/addon/autoinstaller) add-on
 - Open `about:config`
 - set `xpinstall.signatures.required` to `false`
 - Close Firefox Developer Edition
 - `npm run start` to load Firefox Developer Edition with the extension installed
-- `npm run watch` to watch for changes and [reload the extension automatically](https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/jpm#Developing_without_browser_restarts)
 
-**Note: These `npm` commands assume you're on Windows with Firefox Developer Edition installed, modify the paths as needed.**
+**Note: The `npm` command assumes you're on Windows with Firefox Developer Edition installed. Modify the paths as needed.**
```

Comment:
I updated the README a while ago but never pushed it. I've pushed it now so these changes are no longer required.